addons/ekco/template/base/rolebinding-minio.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
can you add this to ekco if you havent already?

https://github.com/replicatedhq/ekco/blob/main/deploy/serviceaccount.yaml

Reply:
done